### ISSUE TITLE
RuntimeError: No type found

### ISSUE THREAD

**Issue by ggpasqualino** [rmosolgo/graphql-ruby#601]
**Current behavior**
When the client sends a query with an invalid type (a typo for example) there is an exception on the server side.

So in the following query
```graphql
query($q: Strng){
  queryArtists(byName: $q){
    id
  }
}
```
we would have a `RuntimeError: No type found 'Strng'`.

**Desired behavior**
The client should get back an error message telling the query is invalid because the type doesn't exist.

For example, if we run the same query on [Spotify](https://spotify-graphql-server.herokuapp.com/graphql?) we get back the following response

```javascript
{
  "errors": [
    {
      "message": "Unknown type \"Strng\". Did you mean \"String\"?",
      "locations": [
        {
          "line": 15,
          "column": 11
        }
      ]
    }
  ]
}
```

**Comment by rmosolgo** [comment]
Thanks for reporting this! I agree, it should be a client error.

**Comment by rmosolgo** [comment]
Taking a crack at it here: #603

**Comment by rmosolgo** [comment]
This will be fixed by 1.5.2 which I'll release today or tomorrow!
